app/Repositories/TodoRepositories.php

```diff
@@ -0,0 +1,32 @@
+<?php
+
+namespace App\Repositories;
+
+use App\Models\Todo;
+use App\Repositories\Interfaces\TodoRepositoryInterface;
+use Illuminate\Database\Eloquent\Collection;
+use Illuminate\Database\Eloquent\Model;
+
+class TodoRepositories implements TodoRepositoryInterface
+{
+    public function index(): Collection|Todo
+    {
+        return Todo::all();
+    }
+
+    public function store($data): Model|Todo
```

Comment:
there isn't data type here. You are using the data as an array. I can provide any data and break your method. In such cases, you need to use dto-class